src/components/calculatorButton.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Please kindly ensure that no linter is disabled, linters are tools that analyze your code to identify and flag potential issues, errors, or stylistic inconsistencies. They are widely used in software development to ensure code quality and maintainability.